tests/s3_tester.c

```diff
@@ -1335,6 +1335,14 @@ int aws_s3_tester_client_new(
     return AWS_OP_SUCCESS;
 }
 
+/* Disable tsan as we hack into the client threaded data */
```

Comment:
😬

Reply:
tried, the client thread is still running, and changing those threaded data from multiple places. 

Maybe better to just suppress tsan